neuralmonkey/decoders/sequence_classifier.py

```diff
@@ -23,18 +23,33 @@ def __init__(self,
                  encoders: List[Any],
                  vocabulary: Vocabulary,
                  data_id: str,
-                 layers: Optional[List[int]]=None,
-                 activation: Callable[[tf.Tensor], tf.Tensor]=tf.tanh,
+                 layers: List[int]=None,
+                 activation_fn: Callable[[tf.Tensor], tf.Tensor]=tf.nn.relu,
                  dropout_keep_prob: float=0.5,
                  save_checkpoint: Optional[str]=None,
```

Comment:
Ty jo, tak tohle optional absolutne netusim jak ma fungovat. Co jsem si k tomu nasel, tak jsem pochopil ze to z toho parametru dela dobrovolny parametr, coz ale pak nefunguje, takze jsem to zrusil a to none mi tam asi zbylo omylem, tak jej take zrusim. Ty layers jsou proste povinny parametr v teto tride.